core/node/api_server/src/tx_sender/master_pool_sink.rs

```diff
@@ -64,7 +64,7 @@ impl TxSink for MasterPoolSink {
     async fn submit_tx(
         &self,
         tx: &L2Tx,
-        execution_metrics: TransactionExecutionMetrics,
+        execution_outputs: &SandboxExecutionOutput,
```

Comment:
Micro-nit: Since the type is named "..Output", it would make sense to name the arg `execution_output` (singular) for consistency.